fetch.bs

```diff
@@ -2123,11 +2125,11 @@ clearly stipulates that <a>connections</a> are keyed on
      WebSocket saner -->
 
 
-<h3 id=http-cache-partitions>HTTP cache partitions</h3>
+<h3 id=http-state-partition>Network partition key</h3>
 
```

Comment:
Let's add a definition here of a network partition key as a tuple consisting of a site and null or an implementation-defined value.

Reply:
Done.